app/services/websocket_server.py

```diff
@@ -22,6 +23,56 @@
 def connect(sid, environ):
     print("connect ", sid)
 
+# get returnTicker, grabs data from tickers table
+async def get_tickers(token_hexstr):
+
+    # init
+    where = ''
+    placeholder_args = []
+
+    # if token is passed in, add where addr = token
+    if token_hexstr:
+        where = '("token_address" = $1)'
+        placeholder_args = [Web3.toBytes(hexstr=token_hexstr), ]
+
+    # connect to db and grab data from tickers table
+    async with App().db.acquire_connection() as conn:
+        return await conn.fetch(
+            """
+            SELECT *
+            FROM tickers
+            WHERE {}
+            ORDER BY token_address DESC
+            """.format(where),
+            *placeholder_args)
+
+# format ticker information to camelcase from underscore, sanitize, add modified date
```

Comment:
Oh, also, suggestion! Python [docstrings](https://www.python.org/dev/peps/pep-0257/) might be a better way to do this